### ISSUE TITLE
Example stacked bar chart doesn't work with Polars

### ISSUE THREAD

**Issue by jonmmease** [hex-inc/vegafusion#286]
Moving https://github.com/altair-viz/altair/issues/3003 from Altair repo, reported by @braaannigan

----
If we try the example stacked bar chart but convert to Polars it doesn't work (I found this with a similar error on my own data):

```python
import altair as alt
from vega_datasets import data
import polars as pl
import vegafusion as vf

vf.enable()

source = pl.from_pandas(data.seattle_weather())

alt.Chart(source).mark_bar(
    cornerRadiusTopLeft=3,
    cornerRadiusTopRight=3
).encode(
    x='month(date):O',
    y='count():Q',
    color='weather:N'
)
```

with error message:

```
thread 'tokio-runtime-worker' panicked at 'assertion failed: `(left == right)`
  left: `LargeUtf8`,
 right: `Utf8`: StringArray expects DataType::Utf8', /root/.cargo/registry/src/github.com-1ecc6299db9ec823/arrow-array-32.0.0/src/array/byte_array.rs:268:9
thread 'tokio-runtime-worker' panicked at 'assertion failed: `(left == right)`
  left: `LargeUtf8`,
 right: `Utf8`: StringArray expects DataType::Utf8', /root/.cargo/registry/src/github.com-1ecc6299db9ec823/arrow-array-32.0.0/src/array/byte_array.rs:268:9
thread 'tokio-runtime-worker' panicked at 'assertion failed: `(left == right)`
  left: `LargeUtf8`,
 right: `Utf8`: StringArray expects DataType::Utf8thread '', tokio-runtime-worker/root/.cargo/registry/src/github.com-1ecc6299db9ec823/arrow-array-32.0.0/src/array/byte_array.rs' panicked at ':assertion failed: `(left == right)`
  left: `LargeUtf8`,
 right: `Utf8`: StringArray expects DataType::Utf8268', :/root/.cargo/registry/src/github.com-1ecc6299db9ec823/arrow-array-32.0.0/src/array/byte_array.rs9:
268:9
thread 'tokio-runtime-worker' panicked at 'assertion failed: `(left == right)`
  left: `LargeUtf8`,
 right: `Utf8`: StringArray expects DataType::Utf8', /root/.cargo/registry/src/github.com-1ecc6299db9ec823/arrow-array-32.0.0/src/array/byte_array.rs:268:9
thread 'tokio-runtime-worker' panicked at 'assertion failed: `(left == right)`
  left: `LargeUtf8`,
 right: `Utf8`: StringArray expects DataType::Utf8', /root/.cargo/registry/src/github.com-1ecc6299db9ec823/arrow-array-32.0.0/src/array/byte_array.rs:268:9
```

**Comment by jonmmease** [comment]
Thanks for the report, I think this may be a manifestation of https://github.com/apache/arrow-rs/issues/2654. I'll look into whether updating Arrow helps (but it may not as this bug is still open), and if not then we can cast `LargeUtf8` to `Utf8` when needed.

**Comment by jonmmease** [comment]
This fix will be released in 1.1.2 shortly

**Comment by jonmmease** [comment]
Fix released in 1.1.2. cc @braaannigan
